operator/backrest/repo.go

```diff
@@ -199,7 +199,15 @@ func UpdateResources(clientset *kubernetes.Clientset, restConfig *rest.Config, c
 	if resource, ok := cluster.Spec.BackrestResources[v1.ResourceMemory]; ok {
 		requestResourceList[v1.ResourceMemory] = resource
 	} else {
```

Comment:
Seems like `limitResourceList[v1.ResourceMemory]` should be set at some point. This block seems like the right place.